test/unit/extra_fields_test.rb

```diff
@@ -1,279 +1,230 @@
 require 'test_helper'
 
-
 class ExtraFieldsTest < ActiveSupport::TestCase
 
-  setup do
-    @provider = FactoryBot.create :provider_account
-    @buyer = FactoryBot.create :buyer_account, :provider_account => @provider
-
-    FactoryBot.create(:fields_definition, :account => @provider, :target => "Account",
-            :name => "provider_extra_field", :required => true)
-    FactoryBot.create(:fields_definition, :account => Account.master,
-            :target => "Account",
-            :name => "master_extra_field", :required => true)
+  def setup
+    @provider = FactoryBot.create(:provider_account, provider_account: master_account)
```

Comment:
Why do we need to set the accound for provider_account?

Reply:
You are right, we don't need it, so I will remove that :smile: